examples/e2ebot/config.yml

```
@@ -13,4 +13,4 @@ pipeline:
     epochs: 200
 policies:
 - name: TEDPolicy
-  epochs: 200
+  epochs: 200
```

not sure what are the guidelines, but all other `yaml` files have blank line in the end
```suggestion
  epochs: 200

```